cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsRdsStatusLookupService.java

```diff
@@ -36,17 +48,25 @@ public ExternalDatabaseStatus getStatus(AuthenticatedContext ac, DatabaseStack d
         DescribeDBInstancesRequest describeDBInstancesRequest = new DescribeDBInstancesRequest().withDBInstanceIdentifier(dbInstanceIdentifier);
         DescribeDBInstancesResult describeDBInstancesResult;
         try {
+            if (preDescribeMessage != null) {
+                LOGGER.debug(preDescribeMessage);
+            }
             describeDBInstancesResult = rdsClient.describeDBInstances(describeDBInstancesRequest);
         } catch (DBInstanceNotFoundException ex) {
-            LOGGER.debug("DB Instance does not exist: {}", ex.getMessage());
-            return ExternalDatabaseStatus.DELETED;
+            LOGGER.debug(notFoundMessage, ex.getMessage());
+            describeDBInstancesResult = null;
```

Comment:
is this ok to to return with null ?

Reply:
Yes. That was the original behavior of `getDescribeDBInstancesResult()` below (lines 70 -- 89). That method will directly return the response of `getDescribeDBInstancesResultInternal()`. There are two other places where this internal stuff is called, `getStatus()` above (lines 30 -- 38) and the new `getActiveSslRootCertificate()` below (lines 64 -- 72), both handle the `null` response correctly. All mentioned methods have been covered with UT, including cases for the `null` response.